assets/javascripts/kitten/components/cards/simple-card/components/image.js

```diff
@@ -1,96 +1,104 @@
 import COLORS from '../../../../constants/colors-config'
 
-const StyledImage = styled.img`
-  width: 100%;
-  display: block;
-`
-const playerButtonSize = pxToRem(70)
-
-const StyledPlayerButton = styled.div`
-  width: ${playerButtonSize};
-  height: ${playerButtonSize};
-  background: ${COLORS.font1};
-  position: absolute;
-  top: calc(50% - ${playerButtonSize} / 2);
-  left: calc(50% - ${playerButtonSize} / 2);
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  z-index: 2;
-`
-
 const StyledImageContainer = styled(Marger)`
   position: relative;
   transition: opacity ease 600ms, z-index ease 600ms;
   z-index: 1;
+  background-color: var(--k-SimpleCard-image-container-background);
+
+  &.k-SimpleCard__imageContainer--ratio {
+    overflow: hidden;
+    position: relative;
+    padding-top: calc(100% / calc(var(--k-SimpleCard-image-container-ratio)));
 
-  background-color: ${({ imageContainerBackground }) =>
-    imageContainerBackground};
+    & > img {
+      position: absolute;
+      top: 0;
+      height: auto;
+      text-align: center;
+    }
+  }
 
-  ${({ imageContainerRatio }) =>
-    imageContainerRatio &&
-    css`
-      overflow: hidden;
-      position: relative;
-      padding-top: calc(100% / calc(${imageContainerRatio}));
+  .k-SimpleCard__playerButton {
+    width: var(--k-SimpleCard-player-button-size);
+    height: var(--k-SimpleCard-player-button-size);
+    background: ${COLORS.font1};
+    position: absolute;
+    top: calc(50% - var(--k-SimpleCard-player-button-size) / 2);
+    left: calc(50% - var(--k-SimpleCard-player-button-size) / 2);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 2;
+  }
 
-      & > img {
-        position: absolute;
-        top: 0;
-        height: auto;
-        text-align: center;
-      }
-    `}
+  .k-SimpleCard__image {
+    width: 100%;
+    display: block;
+  }
 `
 
-export class Image extends PureComponent {
-  render() {
-    const {
-      imageProps,
-      withPlayerButtonOnImage,
-      arrowColor,
-      ariaLabel,
-      imageContainerBackground,
-      imageContainerRatio,
-    } = this.props
-
-    const PlayerButtonOnImage = props => (
-      <StyledPlayerButton>
-        <Text
-          size="default"
-          weight="regular"
-          color={props.arrowColor}
-          aria-label={props.ariaLabel}
+export const Image = ({
+  imageProps,
+  withPlayerButtonOnImage,
+  arrowColor,
+  ariaLabel,
+  imageContainerBackground,
+  imageContainerRatio,
+  className,
+  playerButtonSize,
+  style,
+}) => {
+  return (
+    <StyledImageContainer
+      bottom="2"
+      className={classNames(
+        "k-Card__imageContainer",
+        "k-SimpleCard__imageContainer",
+        className,
+        { 'k-SimpleCard__imageContainer--ratio': imageContainerRatio },
+      )}
+      style={{
+        ...style,
+        '--k-SimpleCard-image-container-background': imageContainerBackground,
```

Comment:
Pas super important : jusque là, les noms des custom properties sont sans le `-k-`, mais c'est pas un problème si tu le laisses

```suggestion
        '--SimpleCard-image-container-background': imageContainerBackground,
```